frontend/javascripts/oxalis/view/right-border-tabs/starting_job_modals.js

```diff
@@ -28,23 +32,26 @@ type StartingJobModalProps = {
   jobName: string,
   description: Node,
   isBoundingBoxConfigurable?: boolean,
+  chooseSegmentationLayer?: boolean,
 };
 
 function StartingJobModal(props: StartingJobModalProps) {
   const isBoundingBoxConfigurable = props.isBoundingBoxConfigurable || false;
+  const chooseSegmentationLayer = props.chooseSegmentationLayer || false;
```

Comment:
The main changes here and below are that I enabled `StartingJobModal` to not only select color layers but also segmentation layers when `chooseSegmentationLayer=true` is passed to the component. The changes are mainly removing "color" from that variable names as the layers the component now works with might also be of kind segmentation.